src/Microsoft.AspNetCore.Blazor.Server/Builder/BlazorApplicationBuilderExtensions.cs

```diff
@@ -38,9 +39,10 @@ public static class BlazorApplicationBuilderExtensions
         /// <summary>
         /// Configures the middleware pipeline to work with Blazor.
         /// </summary>
-        /// <param name="app"></param>
-        /// <param name="options"></param>
-        public static void UseBlazor(
+        /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
+        /// <param name="options">Options to configure the middleware.</param>
+        /// <returns>The <see cref="IApplicationBuilder"/>.</returns>
+        public static IApplicationBuilder UseBlazor(
             this IApplicationBuilder app,
             BlazorOptions options)
```

Comment:
Usually with middleware we just accept the options as-is becuase the options is constructed by the user, and not from DI. `UseBlazor()` would use the options from DI.

`AddBlazor(...)` would take an action, since that's going to compose with DI.